resources/requests.py

```diff
@@ -34,5 +35,24 @@
 ]
 
 class RequestResource(Resource):
-    def get(self):
-        return dtrequest
+    """Get all request"""
+    def get(self):     
+        return dtrequest, 200
```

Comment:
This get method will fetch all the requests regardless of whether they belong to the logged in user as long as they are in the dtrequest dictionary.

Reply:
Ok, I'll fix it asap